docs/Reference/scully-api/findPlugin.md

```diff
@@ -0,0 +1,33 @@
+---
+title: findPlugin
+published: true
+lang: en
+position: 100
+---
+
+# findPlugin
+
+The [`findPlugin`](https://github.com/scullyio/scully/blob/main/libs/scully/src/lib/pluginManagement/pluginConfig.ts#L49) function retrieves a plugin from Scully and returns it. It takes a single parameter:
```

Comment:
It does actually accept 3 parameters! 
So, it takes 1 to 3 params. If the first one is a type, the second is the name (name can be a string, or a symbol). the 3rth one is not really for external use, but you can use that if you want to use optional plugins (you need to set it to false then, you get undefined if the plugin isn't there)
BTW, you can have plugins with the same name for different types.